### PR TITLE
add windows testing bot

### PR DESCRIPTION
**Pull request by devoncarew**
fix https://github.com/dart-lang/dartdoc/issues/511

This will need a bit of configuration once it's landed -

### CODE REVIEW

**Comment by devoncarew** [comment]
[Build status](https://ci.appveyor.com/project/devoncarew/dartdoc)

File: tool/grind.dart [hunk 1]
```diff
@@ -145,6 +145,10 @@ indexResources() {
   out.writeAsString(buffer.toString());
 }
 
+@Task('analyze, test, and self-test dartdoc')
+@Depends(analyze, test, testDartdoc)
+buildbot() => null;
```
Comment (sethladd) [inline tool/grind.dart]: love this

**Comment by sethladd** [comment]
@devoncarew should we rebase and merge this?

**Comment by devoncarew** [comment]
merged -

**Comment by sethladd** [comment]
awesome, thanks!

**Pull request merged**